java/ql/integration-tests/all-platforms/java/buildless-inherit-trust-store/test.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from create_database_utils import *
+from buildless_test_utils import *
+from diagnostics_test_utils import *
+import subprocess
+import os.path
+
+repo_server_process = subprocess.Popen(["python3", "../server.py"], cwd = "repo")
+
+mypath = os.path.abspath(os.path.dirname(__file__))
+certspath = os.path.join(mypath, "jdk8_shipped_cacerts_plus_cert_pem")
+maven_certs_option = "-Djavax.net.ssl.trustStore=" + certspath
+
+try:
+  run_codeql_database_create([], lang="java", extra_args=["--build-mode=none"], extra_env={"MAVEN_OPTS": maven_certs_option, "CODEQL_JAVA_EXTRACTOR_TRUST_STORE_PATH": certspath})
```

Comment:
Does setting `CODEQL_JAVA_EXTRACTOR_TRUST_STORE_PATH` not circumvent the inheritance logic?

Reply:
Yes. It's the best I can do for testing without cloning a JAVA_HOME, which I've tried to do with other tests with ensuing disk-space problems in the past.

Reply:
I am not sufficiently expert in Java tooling to understand why that's the case, but I will take your word for it. Though, if so, it would be good to indicate this somewhere in the test name / comments / etc. so that nobody gets mislead with regards to what this is actually testing.